### PR TITLE
Themed topic page 

### PR DESCRIPTION
**Pull request by hstandaert**
### Teaser-topic
Component has been updated according to the latest designs (https://zpl.io/a8KPw7Q)

### Map
Styling for the drupal map has been moved to the gent-base theme (and removed from styleguide, since dg_maps is a drupal module that is used solely by drupal projects).

Furthermore, the controls have been styled according to the latest designs (https://zpl.io/agNLlkQ)

### Gent-base integration 
Integrated in gent-base. The `teaser` view mode for the following content types have been styled to fit the topic page:

- Topic page (`topic`)
- Topic link (`topic_link`)
- Info page (`infopage`)

## How Has This Been Tested?

Locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [x] I have updated the gent_base CHANGELOG accordingly.
- [x] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig [hunk 1]
```diff
@@ -7,6 +7,18 @@
         'text': teaser_paragraph_text
       } %}
 
+      <ul class="links teaser-topic-links">
+        {% for link in teaser_topic_links %}
+          <li>
+            {% include '@link' with {
+              'text': link.text,
+              'aria_labelledby': link.uid,
```
Comment (delrueba) [inline styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig]: Why are these aria-labelledby?
Link text should be sufficient, with the exception of read-more links*

*regarding read-more links, we can no longer use aria-labelledby as it's not conform WCAG 2.1 2.5.3 
"For user interface components with labels that include text or images of text, the name contains the text that is presented visually."

We'd better replace them by hidden spans, is this feasible in this ticket or would you prefer a bugticket?

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig (hunk 1 above)
Comment (hstandaert) [reply styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig]: Good point. Better to create a bug ticket for this 🙂

File: styleguide/components/71-pages/general/topic-overview-page/topic-overview-page--with-images.twig [hunk 2]
```diff
@@ -35,7 +35,8 @@
             "teaser_link_text": topic.teaser_link_text,
```
Comment (delrueba) [inline styleguide/components/71-pages/general/topic-overview-page/topic-overview-page--with-images.twig]: This is now variable 'read more' / 'show more'
Maybe we should take it out of config altogether, it's not supposed to be configurable.

File: styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig [hunk 3]
```diff
@@ -7,6 +7,17 @@
         'text': teaser_paragraph_text
       } %}
 
+      <ul class="links teaser-topic-links">
```
Comment (delrueba) [inline styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig]: According to design it's either text or links, not both.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: templates/core/block/block--block-news-events-related-to-topic.html.twig [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{#
```
Comment (meirege) [inline templates/core/block/block--block-news-events-related-to-topic.html.twig]: Does this belong in gent_base? Or is it also to project specific?

File: templates/core/fields/field--node--field-url--topic-link.html.twig [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{#
```
Comment (meirege) [inline templates/core/fields/field--node--field-url--topic-link.html.twig]: Same as above for this file?

**Review by meirege — CHANGES_REQUESTED** [review]

File: templates/core/block/block--block-news-events-related-to-topic.html.twig (hunk 4 above)
Comment (delrueba) [reply templates/core/block/block--block-news-events-related-to-topic.html.twig]: This is not the 'related articles' component we have in styleguide?

Comment (hstandaert) [reply templates/core/block/block--block-news-events-related-to-topic.html.twig]: yes, but in this case: it's specifically related to the `topic` content type, so it should be moved into the project 🙂

**Review by meirege — APPROVED** [review]

**Review by delrueba — APPROVED** [review]

**Pull request merged**